src/Test/Wordpress/PluginActionsTest.php

```diff
@@ -61,36 +61,72 @@ public function testLoginWordPressFail()
         $this->assertEquals(false, $response['success']);
     }
 
-    public function testPatchPluginSettings()
+    public function testGetPluginSettings()
```

Comment:
Still need a more descriptive test name.  Something like `testGetPluginSettingsHandlesValidSetting()`.  Need to know what the expected result of the test is.